examples/sparsestream/requirements.txt

```diff
@@ -0,0 +1,3 @@
+fastapi_websocket_pubsub
+tweepy
+deepsparse
```

Comment:
can you set minimum versions for these? definitely deepsparse needs to be recent to have the transformer pipeline, maybe `deepsparse>=0.11`